Sources/AssistiveButton/AssistiveButton.swift

```diff
@@ -96,19 +120,46 @@ final class AssistiveButton: UIButton {
         tapHandler()
     }
 
+    @objc private func hardwareShortcutPressed() {
+        tapHandler()
+    }
+
+    @objc private func panRecognizerHandler(_ recognizer: UIPanGestureRecognizer) {
+        let view = recognizer.view
+        switch recognizer.state {
+            case .began:
+                stopTimer()
+                beginPoint = recognizer.location(in: view)
+            case .changed:
+                recognizer.setTranslation(.zero, in: view)
+                let point = recognizer.location(in: view)
+
+                let offsetX = point.x - beginPoint.x
+                let offsetY = point.y - beginPoint.y
+
+                center.x += offsetX
+                center.y += offsetY
```

Comment:
It is simple enough to be written shorter by my opinion 
`center.x += point.x - beginPoint.x`
`center.y += point.y - beginPoint.y`

Reply:
Yeah I know, but I think current variant is more readable.